### ISSUE TITLE
Subcategory cannot be unselected

### ISSUE THREAD

**Issue by deleteBigTech** [mayeaux/nodetube#267]
**Example:**
![image](https://user-images.githubusercontent.com/64044709/80326669-766f4480-8831-11ea-9fda-9860d94e9cfa.png)

I can change the subcategory but cant leave it empty once i click on some option ... not even after changing category, saving, and changing again to the former category

**Comment by mayeaux** [comment]
Yeah that's a bit tricky. I suppose the best solution to get out the door is when you change the category (or 'refresh' the category) the subcategory is unselected.

**Comment by deleteBigTech** [comment]
@mayeaux it doesnt work

**Comment by mayeaux** [comment]
I mean the best solution to address the problem, it will still have to be coded.

**Comment by deleteBigTech** [comment]
perfect @mayeaux !! thks
